ui/webui/src/components/storage/InstallationDestination.jsx

```diff
@@ -147,6 +153,24 @@ const LocalStandardDisks = ({ onAddErrorNotification }) => {
                         totalDisksCnt
                     )}
                 </Label>
+                <FlexItem align={{ default: "alignRight" }}>
+                    <Button
+                      aria-label={_("Rescan disks")}
+                      id="rescan-disks"
+                      onClick={() => {
+                          scanDevicesWithTask().then(res => {
+                              runStorageTask({
+                                  task: res[0],
+                                  onSuccess: () => resetPartitioning().then(() => setRefreshCnt(refreshCnt + 1), onAddErrorNotification),
```

Comment:
Why do we need the refreshCnt at all? Is it to inform the component to redraw?

Reply:
Yup, its other occurrence in `useEffect(..., [refreshCnt])` means the callback will be triggered when it changes.

Reply:
Exactly! Thanks @VladimirSlavik <3